# HG changeset patch
# User A.R. Burgers <arburgers@gmail.com>
# Date 1553539474 25200
#      Mon Mar 25 11:44:34 2019 -0700
# Node ID 342fc514de46084988167615a0dd362d45886e13
# Parent  392b14b0c085e5c560028a8fc84f15a59cef9ab9
Fix typo in subsasgn error message (bug #55994).

* cdef-object.cc (cdef_object_array::subsagn): Correct "subasgn" to "subsasgn".

diff --git a/libinterp/octave-value/cdef-object.cc b/libinterp/octave-value/cdef-object.cc
--- a/libinterp/octave-value/cdef-object.cc
+++ b/libinterp/octave-value/cdef-object.cc
@@ -452,17 +452,17 @@ namespace octave
             octave_value tmp = obj.subsasgn (type.substr (1), next_idx,
                                              rhs, ignore_copies);
 
             cdef_object robj = to_cdef (tmp);
 
             if (! robj.ok ()
                 || robj.is_array ()
                 || robj.get_class () != get_class ())
-              error ("subasgn: invalid assignment into array of %s objects",
+              error ("subsasgn: invalid assignment into array of %s objects",
                      class_name ().c_str ());
 
             // Small optimization, when dealing with handle
             // objects, we don't need to re-assign the result
             // of subsasgn back into the array.
 
             if (! robj.is (a(0)))
               {
